modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -1,55 +1,31 @@
 = DataWeave System Properties
-
-
-DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties].
-
+DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime:mule-app-properties-system.adoc[System Properties].
```

Comment:
We need to add something about the compatibility flags here and link out to the new versioning doc.

Reply:
Do _all_ the properties that are compatibility flags have "Available for language levels:" versions listed? Asking bc we say that they will expire at some point, but could any of the new properties become compatibility flags in the future? If so, how could we know when those would expire. Maybe I'm overthinking it?